playbooks/provisioning/openstack/sample-inventory/openstack.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
```

Comment:
Ohh, good point! I didn't know it had one. For the record, this is the openstack inventory script that comes from Ansible, not something we wrote or going to maintain ourselves. But it's worth investigating shade's inventory, too.